src/App.js

```diff
@@ -3,8 +3,10 @@ import Homepage from "./components/Homepage"
 import Header from "./components/Header"
 import Jumbotron from './components/Jumbotron'
 import Footer from "./components/Footer"
+import OrderCard from "./components/OrderPage/OrderCard";
```

Comment:
This component isn't being used here so this import can be binned

Reply:
I can't comment on it here, but please also remove the import for `useEffect` since it's also not used here